src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import "./index.scss";
+
+export default function Blogs() {
+  const [blogs, setBlogs] = useState([]);
+  const URL =
+    "https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts";
+  const fetchBlogs = async () => {
+    const data = await fetch(URL);
+    const fetchedBlogs = await data.json();
+    setBlogs(fetchedBlogs);
+    console.log(fetchedBlogs);
+  };
+  useEffect(() => {
+    fetchBlogs();
+  }, []);
+  const NUM_BLOGS_TO_SHOW = 9;
+  const fetchedBlogsArr = blogs.slice(0, NUM_BLOGS_TO_SHOW);
+  console.log(fetchedBlogsArr);
+  return (
+    <div className="blogsMainContainer">
+      <h2 className="ourBlogTitle">Our Blog</h2>
+      <p>
+        Infomration about violence, women and health. Read different topics
+        <br />
+        that contribute positivley to the mental health of women.
+      </p>
+      <SearchBar />
+      <Container>
+        {fetchedBlogsArr.map((blog) => {
+          return (
+            <div className="blogsContainer">
```

Comment:
I think class name is a bit confusing since you have already bootstrap container above this div

Reply:
it's a container that contains the blogs container, if you have any suggestions please let me know them, I can change it to blogsSection if it's much more convenient.

Reply:
yeah, it's more clear. thank you